src/components/DevelopmentGuide/UsingNextjsImage.mdx

```diff
@@ -0,0 +1,480 @@
+import { Meta, Source } from "@storybook/addon-docs/blocks";
+import { Box } from "@chakra-ui/react";
+
+import ComponentDocsHeader from "../../utils/components/ComponentDocsHeader";
+import Image from "../Image/Image";
+import Link from "../Link/Link";
+
+<Meta title="Development Guide/Using Next.js Image" />
+
+<ComponentDocsHeader
+  category="Development Guide"
+  componentName="Using Next.js Image"
+  summary="Recommendations for using the Next.js Image in place of the Reservoir Image"
+/>
+
+## Table of contents
+
+- [Objectives and goals](#objectives-and-goals)
+- [Recommendations](#recommendations)
+  - [Aspect ratio](#aspect-ratio)
+  - [Cropping](#cropping)
+  - [Custom width](#custom-width)
+- [Examples](#examples)
+  - [Dynamic example](#dynamic)
+  - [Component example](#component)
+  - [Proof of concept](#proof-of-concept)
+- [Other useful props](#other-useful-props)
+  - [onError](#onerror)
+  - [placeholder](#placeholder)
+  - [sizes](#sizes)
+
+## Objectives and goals
+
+Based on [Image performance: DC Phase
+2](https://docs.google.com/document/d/1hZsnSOmKFQFtAaNAWjZ9pd44m6wYygn_bUaeDNyrPr0/edit?tab=t.0#heading=h.9wuc905l6iod)
+in the DAM team, there is no perceivable or metric-based reason to use the
+Reservoir `Image` component over the Next.js `Image` component. Conversely,
+using the Next.js `Image` component provides advantages including, but not
+limited to:
+
+- The Next.js `Image` component is meant to optimize image performance within
+  the Next.js framework.
+- The Next.js team works with the Chrome browser team to optimize their
+  components. This means better maintenance and optimization from Next.js that
+  the Reservoir team and NYPL Digital as a whole cannot readily provide.
+- Using the Next.js `Image` component brings NYPL apps deeper into the Next.js
+  ecosystem, which could allow framework-specific optimizations in the future.
+
+While the Next.js `Image` component offers optimization and some enhanced
+styling features, the Reservoir `Image` component has functionality that is not
```

Comment:
Then, I think this can just be "However, the Reservoir `Image` component has..."